plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/flags/KnownFlags.kt

```diff
@@ -0,0 +1,9229 @@
+package org.jetbrains.bazel.languages.bazelrc.flags
+
+// temporary generated from bazel help build.
+// will be followed up with a more comprehensive variant including all the flag and extra data (tags / availability for what commands / etc)
+internal val flags =
+  object {
+// unknown line:                                                            [bazel release 7.3.1]
+// unknown line: Usage: bazel build <options> <targets>
+// unknown line:
+// unknown line: Builds the specified targets, using the options.
+// unknown line:
+// unknown line: See 'bazel help target-syntax' for details and examples on how to
+// unknown line: specify targets to build.
+// unknown line:
+// unknown line: Options that appear before the command and are parsed by the client:
+// a path
+// may be used multiple times
+    @JvmField
+    @Suppress("unused")
+    val distdir =
+      BazelFlag<String>(
+        name = "distdir",
+        description =
+          """
+          Additional places to search for archives before accessing the network to 
+          download them.
+          """.trimIndent(),
+      )
+
+// default: "false"
+    @JvmField
+    @Suppress("unused")
+    val experimentalRepositoryCacheHardlinks =
+      BazelFlag.boolean(
+        name = "experimental_repository_cache_hardlinks",
+        description =
+          """
+          If set, the repository cache will hardlink the file in case of a cache hit, 
+          rather than copying. This is intended to save disk space.
+          """.trimIndent(),
+      )
+
+// an integer
+// default: "0"
```

Comment:
would be great to use this info too, but that can be done in a subsequent PR :)

Reply:
ooo i plan to definitely use that :) but waiting to have it done would have made this big and late. so going bit by bit